test/browser/generic/tabular.js

```diff
@@ -4,66 +4,66 @@ const tf = require('@tensorflow/tfjs-core');
 describe('Encodings', function () {
   describe('LabelEncoder', function () {
     it('test the label encoding on array', async function () {
-      const data = [ 'dog', 'cat', 'man', 'dog', 'cat', 'man', 'man', 'cat' ];
-      const encode = new datacook.Encoder.LabelEncoder(data);
-      const fit_data = [
+      let data = ['dog', 'cat', 'man', 'dog', 'cat', 'man', 'man', 'cat'];
+      let encode = new datacook.Encoder.LabelEncoder(data);
+      let fit_data = [
```

Comment:
The above should be constants, why does this change?

Reply:
probabiliy a version issue